tests/e2e/ctl_v3_endpoint_test.go

```diff
@@ -15,26 +15,9 @@
 package e2e
 
 import (
-	"context"
-	"fmt"
-	"net/url"
-	"testing"
-	"time"
-
-	"go.etcd.io/etcd/client/v3"
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
-func TestCtlV3EndpointHealth(t *testing.T) { testCtl(t, endpointHealthTest, withQuorum()) }
-func TestCtlV3EndpointStatus(t *testing.T) { testCtl(t, endpointStatusTest, withQuorum()) }
-func TestCtlV3EndpointHashKV(t *testing.T) { testCtl(t, endpointHashKVTest, withQuorum()) }
-
-func endpointHealthTest(cx ctlCtx) {
-	if err := ctlV3EndpointHealth(cx); err != nil {
-		cx.t.Fatalf("endpointStatusTest ctlV3EndpointHealth error (%v)", err)
-	}
-}
-
 func ctlV3EndpointHealth(cx ctlCtx) error {
```

Comment:
Is this function used anywhere?

Reply:
It 's  used in https://github.com/etcd-io/etcd/blob/main/tests/e2e/ctl_v3_auth_test.go#L1103
I will delete it after `e2e/ctl_v3_auth_test.go` migrated.

Reply:
Another  method:  I move this function to `e2e/ctl_v3_auth_test.go` and delete `tests/e2e/ctl_v3_endpoint_test.go` ?
Do you agree？

Reply:
sgtm

Reply:
Done